Bindicator.py

```diff
@@ -7,27 +7,53 @@
 
 import json
 import Monash
+import BlinkPatterns
+
+address = '3/36 Panorama Street, Clayton 3168' #Replace your address to get geolocationid (Monash Council only for now)
+gbit = GlowBitController.GlowBitController(config["glowbit"]) #Global gbit
```

Comment:
Why did you make this global?

Reply:
gbit is used in another func

Reply:
please move that Function to Monash.py and put this back into startDebug()